<ui version="4.0" >
 <class>KwEdit</class>
 <widget class="QDialog" name="KwEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>KawaiiGL Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs" >
     <property name="acceptDrops" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Model</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>3</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="ProgTextEdit" name="ed" >
         <property name="font" >
          <font>
           <family>Courier</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="html" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Courier'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="tabStopWidth" >
          <number>40</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addFaceBot" >
           <property name="maximumSize" >
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Add</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Vertex Shader</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>3</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="ProgTextEdit" name="vtxEd" >
         <property name="font" >
          <font>
           <family>Courier</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="html" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Courier'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8pt;">&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="tabStopWidth" >
          <number>40</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Fragment Shader</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>3</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="ProgTextEdit" name="fragEd" >
         <property name="font" >
          <font>
           <family>Courier</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="tabStopWidth" >
          <number>40</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="shaderControl" >
     <layout class="QVBoxLayout" name="verticalLayout_6" >
      <property name="leftMargin" >
       <number>3</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>3</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <property name="topMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="update_shader" >
          <property name="maximumSize" >
           <size>
            <width>70</width>
            <height>23</height>
           </size>
          </property>
          <property name="text" >
           <string>Compile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reloadBot" >
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Reload the last program loaded</string>
          </property>
          <property name="text" >
           <string>Reload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="infoLabel" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="shaderEnable" >
          <property name="text" >
           <string>Enable Shaders</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="runTypeBox" />
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addParam" >
          <property name="maximumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="statusTip" >
           <string>Add Parameter</string>
          </property>
          <property name="text" >
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ProgTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
